core-web/libs/portlets/edit-ema/portlet/src/lib/components/dot-ema-dialog/dot-ema-dialog.component.html

```diff
@@ -40,7 +39,7 @@
 
     <p-dialog
         (visibleChange)="closeCompareDialog()"
-        [visible]="this.$compareData()"
+        [visible]="!!$compareData()"
```

Comment:
That name and its use, plus the negation,  make me think hard.

Reply:
Yeah is not clear what's going on. I will look for a better approach. I changed it because the eslint was faling there, because `$compareData()` is an object rather than a boolean

Reply:
Agreed with Arca here.